sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileScan.scala

```diff
@@ -42,6 +44,21 @@ abstract class FileScan(
     false
   }
 
+  override def description(): String = {
+    def seqToString(seq: Seq[Any]) = seq.mkString("[", ", ", "]")
```

Comment:
It seems that we don't need to declare this one-time function. It's different from `FileSourceScanExec` which reuses this function multiple times.